src/mixer/basetrackplayer.cpp

```diff
@@ -720,8 +727,13 @@ void BaseTrackPlayerImpl::slotWaveformZoomSetDefault(double pressed) {
         return;
     }
 
-    double defaultZoom = m_pConfig->getValue(ConfigKey("[Waveform]","DefaultZoom"),
-        WaveformWidgetRenderer::s_waveformDefaultZoom);
+    double defaultZoom = m_pConfig->getValue(ConfigKey("[Waveform]", "DefaultZoom"),
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+            WaveformWidgetRenderer::s_waveformDefaultZoom);
+#else
+            // TODO: This should not be hardcoded.
```

Comment:
I thought about it when I wrote the comments, but opted for `TODO` because that way you can grep for `FIXME` before tagging a release. And in this case the issue needs to before a Qt6 release, not *any* release (i didn't want to block 2.4.0).

If you prefer FIXME I can change that of course.

Reply:
Makes sense to me to leave it as is then :+1: We have to check all the Qt6 version guards anyway before releasing a Qt6 version.